detection_rules/rule.py

```diff
@@ -230,6 +235,11 @@ def ast(self):
         if validator is not None:
             return validator.ast
 
+    def get_beats_types(self) -> List[str]:
```

Comment:
do you think this makes more sense on the query validator?
i don't think QueryRuleData is in the hierarchy for everything?

Reply:
2e26e0f